packages/stable/src/api/annotations/annotationsApi.ts

```diff
@@ -36,6 +36,8 @@ export class AnnotationsAPI extends BaseResourceAPI<AnnotationModel> {
 
   /**
    * [Suggest annotations](https://docs.cognite.com/api/playground/#operation/annotationsSuggest)
+   * 
+   * const created = await client.annotations.create(annotations);
```

Comment:
what's the purpose of this example if we don't show the actual input object?
we run TS compiler here, so it will fail as the variable don't exist